builder/uploader/jenkins_upload.py

```diff
@@ -18,12 +39,13 @@ def update(config, view_name, view_xml):
         get_url = config['url'] + '/view/%s' % view_name
         create_url = config['url'] + '/createView?name=%s' % view_name
         update_url = get_url + '/config.xml'
-        response = requests.get(get_url,
-                                auth=(config['user'], config['password']),
-                                verify=False)
-        if response.status_code != 200:
-            print "Creating view %s" % view_name
-            post(create_url, view_xml, config)
-        else:
-            print "Updating view %s" % view_name
-            post(update_url, view_xml, config)
+
+        do_get = partial(requests.get, get_url, verify=False)
+        response = krb_request(do_get, config)
+        if response is not None:
```

Comment:
Same as the above comment
https://github.com/michalvanco/jenkins-view-builder/pull/2/files#r42723546